archunit/src/main/java/com/tngtech/archunit/core/domain/JavaMethod.java

```diff
@@ -125,6 +126,12 @@ public String getDescription() {
         return "Method <" + getFullName() + ">";
     }
 
+    @PublicAPI(usage = ACCESS)
+    public boolean isOverridden() {
+        Method method = methodSupplier.get();
+        return Arrays.asList(method.getDeclaringClass().getSuperclass().getDeclaredMethods()).contains(method);
```

Comment:
It would be nice to avoid the (expensive) reflection API, but more importantly: Does this even work for you? Please provide a unit test that proves the correctnes of your code.

I'd use something like this:
```java
public class JavaMethodTest {
    @Test
    public void isOverridden() {
        class Base {
            void method1() {}
            void method1(int x) {}
        }
        class Child extends Base {
            void method1() {}
            void method2() {}
        }
        JavaClass childClass = new ClassFileImporter().importClass(Child.class);
        assertThat(childClass.getMethod("method1").isOverridden()).isTrue();
        assertThat(childClass.getMethod("method1", int.class).isOverridden()).isFalse();
        assertThat(childClass.getMethod("method2").isOverridden()).isFalse();
}
```
Your implementation however gives me `childClass.getMethod("method1").isOverridden() == false`.